ui/README.md

```diff
@@ -1,30 +1,31 @@
 # Building required modules to run from shell/IDE
 
 Before running either from console or from IDE you have to manually build:
+- run `npm install` in `ui/client` (only if you want to test/compile FE)
```

Comment:
It should be `npm ci` to be sure that you have the same packages as on CI

Reply:
:+1: true, done